packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -1,12 +1,38 @@
 import { declare } from "@babel/helper-plugin-utils";
 import { addDefault, isModule } from "@babel/helper-module-imports";
 import { types as t } from "@babel/core";
+import getTargets, {
+  filterItems,
+  isRequired,
+} from "@babel/helper-compilation-targets";
+import corejs2Polyfills from "@babel/compat-data/corejs2-built-ins";
+import corejs3Polyfills from "core-js-compat/data";
 
 import getCoreJS2Definitions from "./runtime-corejs2-definitions";
 import getCoreJS3Definitions from "./runtime-corejs3-definitions";
 import { typeAnnotationToString } from "./helpers";
 import getRuntimePath from "./get-runtime-path";
 
+// corejs sometimes lumps multiple functions/identifiers under one name
```

Comment:
Can you add a comment that this part should be align to [packages/babel-preset-env/src/polyfills/corejs3/built-in-definitions.js](https://github.com/babel/babel/blob/master/packages/babel-preset-env/src/polyfills/corejs3/built-in-definitions.js)?

We can move these definitions to `@babel/compat-data` as, in this case, it is reusabled by both `preset-env` and `transform-runtime`.

Reply:
Done.